src/main/resources/public/css/stylesheet.css

```diff
@@ -51,30 +51,33 @@ body {
     color: #3b3d44;
 }
 
-input, textarea {
+.book-form>input,
+textarea {
```

Comment:
don't know if that .book-form> is applying to textarea, same thing a couple places in the file

also I see the selector for textarea in multiple places in this file, is that intentional?

Reply:
you are correct, the .book-form> would not have applied to textarea. Also i have the textarea twice because i wanted to abstract out common rules so that i don't have to repeat them everywhere